packages/gatsby-link/src/types.ts

```diff
@@ -0,0 +1,42 @@
+import { CSSProperties, Ref } from "react"
+import { NavigateOptions, LinkProps } from "@reach/router"
+
+/* eslint-disable @typescript-eslint/interface-name-prefix */
```

Comment:
nit, can you just make this the `eslint-disable-next-line` and put it above `interface Window` ?